src/Features/CSharp/Portable/CodeLens/CSharpCodeLensMemberFinder.cs

```diff
@@ -83,5 +84,41 @@ public override void VisitRecordDeclaration(RecordDeclarationSyntax node)
             _memberBuilder.Add(new CodeLensMember(node, node.Identifier.Span));
             base.VisitRecordDeclaration(node);
         }
+
+        public override void VisitDelegateDeclaration(DelegateDeclarationSyntax node)
+        {
+            _memberBuilder.Add(new CodeLensMember(node, node.Identifier.Span));
+        }
+
+        public override void VisitEnumMemberDeclaration(EnumMemberDeclarationSyntax node)
+        {
+            _memberBuilder.Add(new CodeLensMember(node, node.Identifier.Span));
+        }
+
+        public override void VisitFieldDeclaration(FieldDeclarationSyntax node)
+        {
+            foreach (var variable in node.Declaration.Variables)
+            {
+                _memberBuilder.Add(new CodeLensMember(variable, variable.Identifier.Span));
+            }
+        }
```

Comment:
1. what's teh consequence of this in VS for multi-declarator members?
2. does thsi light up code-lens for fields for *all* users in VS.  if so, i think this should be gated behind a user option peopel can opt into.

Reply:
This has no impact on Visual Studio. This is a VS Code feature.

Reply:
> I'm not sure how to test the UI in VS Code

@dibarbet  can you guide sam on this?

Reply:
> I'm not sure how to test the UI in VS Code, but I would expect it to be able to handle this since today it's already possible to declare multiple properties on the same line of code.

If you mean seeing the actual code lens items inside vscode, it should be relatively straightforward.  Basically:
1.  Make sure the C# extension is installed and you can open a C# project
2.  Point the C# extension to your locally built server as described here - https://github.com/dotnet/vscode-csharp/blob/main/CONTRIBUTING.md#using-a-locally-developed-roslyn-server
3.  Reload the window

Reply:
It's going to end up working like this:
![image](https://github.com/dotnet/roslyn/assets/1408396/621544b9-d0ee-4147-8ad6-7e767596ae7f)